cypress/integration/Crosscutting/PublicUseTest/index.js

```diff
@@ -0,0 +1,31 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('user is navigating to {string}', (a) => {
+    cy.visit(a);
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // we expect a 3rd party library error with message 'list not defined'
+        // and don't want to fail the test so we return false
+        if (err.message.includes('Cannot read property')) {
```

Comment:
Could you provide the entire error message here? Cause we only want to ignore that one known error FOR NOW, but still catch anything else that shows up

Reply:
updated with full error